recipes/pvxslibs/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pvxslibs" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 46b0c31aa8e47813dc409d58067acf637650cdcf2ed415cc2c3a8cecf8231e4d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```

```suggestion
build:
  number: 0
  script: {{ PYTHON }} -m pip install . -vv
  rpaths:
    - lib/
  # epicscorelibs are in site-packages instead of with other shared objects
    - {{ SP_DIR }}/epicscorelibs/lib/
```
You need to tell conda-build if you are linking to shared objects in non-standard places (i.e. inside site-packages).

---

OK, this resolved the Linux build at least. Thanks